SPEC.md

```diff
@@ -565,7 +565,10 @@ JSON Schema for the Container Runtime Manifest (container manifest)
     "apps": [
         {
             "app": "example.com/reduce-worker-1.0.0",
-            "imageID": "sha512-..."
+            "imageID": "sha512-...",
+            "mountPoints": [
+                 {"source": "work", "destination": "work"}
```

Comment:
+1 on this change. Regarding schemas however, small suggestion: make this a more obvious junction table (use name-matched references * cough * foreign keys if you will):
```
"mounts": [
    {"volume": "database", "mountPoint": "db"},
    {"volume": "buildoutput", "mountPoint": "out"}
],
```